<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8J9VN6K" failures="1" tests="5" name="Test" time="41.276" errors="0" timestamp="2024-09-12T18:32:38 IST">
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="2.256"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="11.178"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="2.525">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:58)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="11.417"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="4.027"/>
</testsuite> <!-- Test -->
